src/model/ChillerElectricEIR.hpp

```diff
@@ -60,8 +60,7 @@ namespace model {
    *\param EItoCORFofT Electric Input to Cooling Output Ratio as a Function of Temp
    *\param EItoCORFofPLR Electric Input to Cooling Output Ratio as a Function of Partial Load Ratio
    */
-    explicit ChillerElectricEIR(const Model& model, const CurveBiquadratic& CCFofT, const CurveBiquadratic& EItoCORFofT,
-                                const CurveQuadratic& EItoCORFofPLR);
+    explicit ChillerElectricEIR(const Model& model, const CurveBiquadratic& CCFofT, const CurveBiquadratic& EItoCORFofT, const Curve& EItoCORFofPLR);
```

Comment:
I think the larger discussion is whether it wouldn't make sense, since it requires breaking API, to make all of these be `Curve` objects... So that you could technically pass an `OS:Table:MultiVariableLookUp` (`Table:IndependentVariable` in E+)

Reply:
I support changing these to `Curve` and generally trying to use the base `Curve` type going forward. It just seems to be the most future proof to me.

Reply:
@joseph-robertson:  @kbenne and I discussed this briefly, I think we agree that: 

1. We're breaking API anyways (and that's fine)
2. Might as well change all ctor arguments (and matching getters) to be a Curve object (base class).
3. Document that in the future release notes

Reply:
@jmarrec's plan sounds good to me.

Reply:
(I didn't see you had already answered 2 minutes before me, my bad)

Reply:
On #2 above, change setter arguments too (since ctor uses them).